[2024-05-04T11:50:17.027+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-04T11:50:17.353+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: emaildag.testingfunc scheduled__2024-05-02T00:00:00+00:00 [queued]>
[2024-05-04T11:50:17.596+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: emaildag.testingfunc scheduled__2024-05-02T00:00:00+00:00 [queued]>
[2024-05-04T11:50:17.618+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-05-04T11:50:18.049+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): testingfunc> on 2024-05-02 00:00:00+00:00
[2024-05-04T11:50:18.167+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'emaildag', 'testingfunc', 'scheduled__2024-05-02T00:00:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/emaildag.py', '--cfg-path', '/tmp/tmp8mngm0wm']
[2024-05-04T11:50:18.198+0000] {standard_task_runner.py:91} INFO - Job 50: Subtask testingfunc
[2024-05-04T11:50:18.187+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=571) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-04T11:50:18.200+0000] {standard_task_runner.py:63} INFO - Started process 591 to run task
[2024-05-04T11:50:28.281+0000] {cli_action_loggers.py:137} WARNING - Failed to log action (psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2024-05-04T11:50:54.491+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 50 for task testingfunc ((psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8); 591)
[2024-05-04T11:51:35.678+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-04T11:51:36.320+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-04T11:51:36.411+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
